azurecontainerapps.ts

```diff
@@ -233,12 +230,46 @@ export class azurecontainerapps {
         // Set deployment location, if provided
         let location: string = this.toolHelper.getInput('location', false);
 
-        // If no location was provided, use the default location for the Container App service
-        if (this.util.isNullOrEmpty(location)) {
-            location = await this.appHelper.getDefaultContainerAppLocation();
+        if (!this.util.isNullOrEmpty(location)) {
+            return location;
+        }
+
+        // If no location was provided, attempt to discover the location of the existing Container App Environment linked to the Container App
+        // or Container App Environment provided in the resource group or use the default location.
+        let resourceGroup: string = this.toolHelper.getInput('resourceGroup', false);
+
+        // Get the resource group if it was provided, or generate it from the Container App name
+        !this.util.isNullOrEmpty(resourceGroup) ? resourceGroup : `${this.containerAppName}-rg`;
```

Comment:
This line does not assign the values anywhere. I believe that the `this.toolHelper.getInput` method has an argument to give a default value. Should we use that argument to give `${this.containerAppName}-rg`?

Reply:
+1

Reply:
Addressed this in my latest commit - https://github.com/Azure/container-apps-deploy-action/pull/76/commits/4faa46488fafdd70f3fbfed2a361df52a3c1c0df